static/js/lib/queries/applications.js

```diff
@@ -126,5 +128,18 @@ export default {
         "X-CSRFTOKEN": getCookie("csrftoken")
       }
     }
+  }),
+  applicationUploadResumeQuery: (
```

Comment:
Convention is to call non-GET query configs "mutations" instead of "queries". So this should be `applicationUploadResumeMutation`